web/assets/css/production.less

```diff
@@ -232,6 +232,11 @@ li.prep-status {
   line-height: 28px;
 }
 
+.step-body .dictionary {
+  table-layout: fixed;
+  width: 50%;
```

Comment:
Maybe this should be a `max-width` for cases where the metadata is actually short (no need to push the step output over).